docs/troubleshooting.rst

```diff
@@ -38,6 +38,63 @@ Windows Command Prompt::
 Then run your Python script in the same terminal.
 
 
+------------------------------------------------
+Handling multiple devices with same Service UUID
```

Comment:
I think this applies to multiple devices in general, not just ones with the same service UUID.

Suggested title: Connecting to multiple devices at the same time.